`SystemTests.test_e101_skip_innocuous` test fails with tarball from pypi because of missing tox.ini

The tarball from pypi doesn't contain tox.ini, but one test expects it and fails when it doesn't find it.

```
%: pytest
...
=================================== FAILURES ===================================                                                              
_____________________ SystemTests.test_e101_skip_innocuous _____________________
                                    
self = <test.test_autopep8.SystemTests testMethod=test_e101_skip_innocuous>
                                                                                                                                                  
        def test_e101_skip_innocuous(self):                                                                                                       
            # pep8 will complain about this even if the tab indentation found   
            # elsewhere is in a multiline string. If we don't filter the innocuous                                                                
            # report properly, the below command will take a long time.                                                                           
            p = Popen(list(AUTOPEP8_CMD_TUPLE) +                         
                      ['-vvv', '--select=E101', '--diff',                                                                                         
                       '--global-config={}'.format(os.devnull),        
                       os.path.join(ROOT_DIR, 'test', 'e101_example.py')],                        
                      stdout=PIPE, stderr=PIPE)                                                                                                   
            output = [x.decode('utf-8') for x in p.communicate()][0]     
            setup_cfg_file = os.path.join(ROOT_DIR, "setup.cfg")                                                                                  
            tox_ini_file = os.path.join(ROOT_DIR, "tox.ini")      
            expected = """\                                                                                                                       
    read config path: /dev/null                                                                                                                   
    read config path: {}                                                                                                                          
    read config path: {}                                                 
    """.format(setup_cfg_file, tox_ini_file)
>           self.assertEqual(expected, output)
E           AssertionError: 'read[69 chars]etup.cfg\nread config path: /builddir/autopep8-1.7.0/tox.ini\n' != 'read[69 chars]etup.cfg\n'
E             read config path: /dev/null
E             read config path: /builddir/autopep8-1.7.0/setup.cfg
E           - read config path: /builddir/autopep8-1.7.0/tox.ini

test/test_autopep8.py:1058: AssertionError
```


## Your Environment
* Python version: 3.10.6
* autopep8 version: 1.7.0
* Platform: linux

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -12,6 +12,7 @@ include test/example.py
 include test/iso_8859_1.py
 include test/fake_configuration/.pep8
 include test/fake_pycodestyle_configuration/tox.ini
+include tox.ini
 recursive-exclude test/suite *.py
 recursive-exclude test/suite/out *.py
 exclude .travis.yml
@@ -27,5 +28,4 @@ exclude test/acid_pypi.py
 exclude test/suite
 exclude test/suite/out
 exclude test/vim_autopep8.py
-exclude tox.ini
 exclude update_readme.py

--- a/tox.ini
+++ b/tox.ini
@@ -8,5 +8,5 @@ commands=
     python test/acid.py --aggressive test/example.py
     python test/acid.py --compare-bytecode test/example.py
 deps=
-    pycodestyle>=2.0
+    pycodestyle>=2.9.1
     pydiff>=0.1.2

